### PR TITLE
Add levelbuilder links to project template levels.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Per request from @bakerfranke. Note the name links to the level, the "[E]" links to the level edit page.

image

### CODE REVIEW

File: dashboard/app/views/levels/_admin.html.haml [hunk 1]
```diff
@@ -42,6 +42,11 @@
                   - @level.properties['pages'].each do |page|
                     - page['levels'].each do |level_name|
                       %li= link_to level_name, level_path(Level.find_by_name(level_name))
+            - if @level.properties['project_template_level_name']
```
Comment (ashercodeorg) [inline dashboard/app/views/levels/_admin.html.haml]: Is there a (good) way to save the value of `@level.properties['project_template_level_name']` in a "local HAML variable"?

Comment (Hamms) [reply dashboard/app/views/levels/_admin.html.haml]: can you not just do `- template_name = @level.properties['project_template_level_name']`?

**Comment by ashercodeorg** [comment]
Right. Oops. Thanks.

File: dashboard/app/views/levels/_admin.html.haml [hunk 2]
```diff
@@ -42,6 +42,12 @@
                   - @level.properties['pages'].each do |page|
                     - page['levels'].each do |level_name|
                       %li= link_to level_name, level_path(Level.find_by_name(level_name))
+            - if project_template_level_name = @level.properties['project_template_level_name']
+              - project_template_level = Level.find_by_name(project_template_level_name)
```
Comment (ashercodeorg) [inline dashboard/app/views/levels/_admin.html.haml:46]: It isn't immediately obvious to me whether we want to gate the `li` creation on the level existing. Failing fast and hard seems like a reasonable route.

Comment (Hamms) [reply dashboard/app/views/levels/_admin.html.haml:46]: agreed

**Pull request merged**